index.html

```diff
@@ -0,0 +1,327 @@
+                    </h2>
+
+                    <ul class="grid">
+                        <li class="item">html</li>
+                        <li class="item">css</li>
+                        <li class="item">javascript</li>
+                    </ul>
+                    <div class="card-button ">
+                        <button type="button" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        
+
+          <div class="work-card layout">
+            <div class="Work-card-head ">
+                <div class="Work-card-body">
+                    <div class="Work-card-img4 ">
+                    </div>
+                    <h3 class="work-card-title ">Multi-Post<br>Stories</h3>
+                    <div class="work-card-dot ">
+                        <div class="card-text-main">CANOPY</div>
+                        <div class="work-card-dot-s"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">Back End Dev</div>
+                        <div class="work-card-dot-em2"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">2015</div>
+                    </div>
+                    <h2 class="card-subtext ">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+
+                    <ul class="grid">
+                        <li class="item">html</li>
+                        <li class="item">css</li>
+                        <li class="item">javascript</li>
+                    </ul>
+                    <div class="card-button ">
+                        <button type="button" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="About ">
+            <div class="Aboutme-head ">
+                <div class="Aboutme-body ">
+                    <div class="main-title ">
+                        <h3 class="body-title">
+                            About<br>Myself</h3></div>
+            <p class="About-text ">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h5 class="mini-text-abt ">LET’S CONNECT</h5>
```

Comment:
- [ ] Please always try to stick to the design as much as possible because It will make you more prepared to work in future environments where you will be required to implement Figma designs into working projects. so you should match your design with the given design in project requirements like font-colors,margins,padding, interactions ,etc.

Your design:

![image](https://user-images.githubusercontent.com/90524466/170781501-6a362455-a524-49d5-908c-291cb8afb2b9.png)


Figma design:

![image](https://user-images.githubusercontent.com/90524466/170781636-711780bb-561c-401a-b1ee-39a33822b6b3.png)